index.css

```diff
@@ -0,0 +1,158 @@
+body {
```

Comment:
Overall good work with Twitter clone. Few differences that I noticed are: 
1. Page does not respond well when re-sizing browser window size. Original Twitter website put white part on top of blue part after you re-size the page. This can be achieved by using flex. (Grid is still fine in most cases though) 
2. Some color differences which is not a big deal at all. That blue reminded me of FB blue a little lol 
3. Background bird need to be centered just like the real site. 
Other than above-mentioned feedback, everything looks good. Good work and keep coding!